bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/gnirs/GnirsImaging.scala

```diff
@@ -40,7 +40,7 @@ case class GnirsImaging(blueprint:SpGnirsBlueprintImaging) extends GnirsBase[SpG
     case _              => oldFilter
   }
 
-  include(16, 17, 18, 19, 20, 21) in TargetGroup
+  include(16, 17, 18, 19, 20, 21, 31) in TargetGroup
```

Comment:
Why not 39?

Reply:
D'oh, you're right.  Thanks for pointing this out!